packages/web/src/ui/views/viewer.js

```diff
@@ -80,10 +82,33 @@ const viewer = (state, i18n) => {
           delta[0] -= x;
           delta[1] -= y;
         }
-        if(!timer) timer = requestAnimationFrame(doRotatePan);
+        if(!timer) {
+          cancelAnimationFrame(renderTimer);
```

Comment:
I am making sure that updateAndRender is called only once in an animationFrame. cancel is called because doRotatePanZoom function will call updateAndRender